pkg/execution/debounce/debounce.go

```diff
@@ -307,6 +355,14 @@ func (d debouncer) StartExecution(ctx context.Context, di DebounceItem, fn innge
 
 	switch res {
 	case 0, 1:
+		metrics.IncrQueueDebounceOperationCounter(ctx, metrics.CounterOpt{
```

Comment:
probably good to add this metric on `default` case as well to provide better visibility with a different `op` value.